src/casync-tool.c

```diff
@@ -155,6 +156,11 @@ static void help(void) {
                program_invocation_short_name);
 }
 
+static void version(void) {
```

Comment:
I try to avoid invocating exit() unless there's a really good reason to have it. I think it's much nicer and cleaner to return to the caller, and propagate the exit condition all the way up through all stack frames, and then return cleanly from the main() function, cleaning up everything that needs cleaning up. Using exit() means bypassing clean-up code, and that's dirty.

Hence, no, this neither should have exit() nor be a noreturn function...